### PR TITLE
Refactor EnvSelection to not use mui

### PR DESCRIPTION
**Pull request by sven-urbanski-freiheit-com**
Before: 
image

Now:
image

### CODE REVIEW

File: services/frontend-service/src/ui/components/ServiceLane/EnvSelectionDialog.tsx [hunk 1]
```diff
@@ -56,8 +55,12 @@ export const EnvSelectionDialog: React.FC<EnvSelectionDialogProps> = (props) =>
...
 
     return (
-        <Dialog open={props.open}>
-            <DialogTitle id="alert-dialog-title">{'Select all environments to be removed:'}</DialogTitle>
+        <ConfirmationDialog
+            onConfirm={onConfirm}
+            onCancel={onCancel}
+            open={props.open}
+            headerLabel={'Select all environments to be removed:'}
+            confirmLabel={'Remove environments from app'}>
```
Comment (hannesg) [inline services/frontend-service/src/ui/components/ServiceLane/EnvSelectionDialog.tsx:63]: I think `Remove application from environments` is the correct action.

**Review by hannesg — APPROVED** [review]
I'm mildly shocked that this didn't break any test :|

**Comment by sven-urbanski-freiheit-com** [comment]
> I'm mildly shocked that this didn't break any test :|

It did actually break tests, but adding `test-button-confirm` and `cancel` fixed it. @hannesg

File: services/frontend-service/src/ui/components/ServiceLane/EnvSelectionDialog.tsx (hunk 1 above)
Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/components/ServiceLane/EnvSelectionDialog.tsx]: done

**Comment by hannesg** [comment]
> > I'm mildly shocked that this didn't break any test :|
> 
> It did actually break tests, but adding `test-button-confirm` and `cancel` fixed it. @hannesg

Uh, nice then actually!

**Pull request merged**